### PR TITLE
Merge avidal/master

### PR DESCRIPTION
**Pull request by jeremywiebe**
## Summary:
* Use the stable rust toolchain instead of a specific version (#18)
* Remove rust-toolchain

Issue: XXX-XXXX

## Test plan:

### CODE REVIEW

**Comment by avidal** [comment]
Need any help with this? One of the really annoying parts of Go is that forks can't easily contribute back to each other due to import paths changing.

**Comment by avidal** [comment]
Also, and you may know this already, if you pull in my dictionary impl then you can replace the `kv` code that I hacked into khanacademy.dev for published content versions (and i think for static versions?). The dictionary implementation in fastlike.dev is designed to support official dictionaries in Fastly Compute.

File: xqd_request.go [hunk 1]
```diff
@@ -13,84 +13,84 @@ import (
...
-		return int32(XqdErrInvalidHandle)
+		return XqdErrInvalidHandle
 	}
 
-	return int32(XqdStatusOK)
+	return XqdStatusOK
 }
 
-func (i *Instance) xqd_req_cache_override_set_v2(handle int32, tag int32, ttl int32, swr int32, sk int32, sk_len int32) int32 {
+func (i *Instance) xqd_req_cache_override_v2_set(handle int32, tag int32, ttl int32, swr int32, sk int32, sk_len int32) int32 {
```
Comment (jeremywiebe) [inline xqd_request.go:51]: Matches the Fastly fn name more closely

**Comment by jeremywiebe** [comment]
Thanks for the offer @avidal    I think the PR is ready, so if you feel like taking a look, that'd be great. :)

**Comment by avidal** [comment]
👍 if it merges cleanly it should be good to go

**Review by jaredly — APPROVED** [review]
oh sorry for sitting on this so long -- looks good to me!

**Pull request merged**